Sources/StreamVideo/WebRTC/WebRTCClient.swift

```diff
@@ -120,10 +120,23 @@ class WebRTCClient: NSObject {
     private(set) var callSettings = CallSettings()
     private(set) var videoOptions = VideoOptions()
     private let environment: WebSocketClient.Environment
+    private let apiKey: String
     
+    private var migratingSignalService: Stream_Video_Sfu_Signal_SignalServer?
+    private var migratingWSClient: WebSocketClient?
+    private var migratingToken: String?
+    private var fromSfuName: String?
+    private var temp: PeerConnection?
+
     var onParticipantsUpdated: (([String: CallParticipant]) -> Void)?
     var onSignalConnectionStateChange: ((WebSocketConnectionState) -> ())?
     var onParticipantCountUpdated: ((UInt32) -> ())?
+    var onSessionMigrationEvent: (() -> ())? {
```

Comment:
Who will be setting this?

Reply:
It's set when the webrtc client is being setup, by the `CallController`.